admin-dev/themes/new-theme/package.json

```diff
@@ -22,6 +22,7 @@
     "eonasdan-bootstrap-datetimepicker": "^4.17.47",
     "events": "^3.0.0",
     "fos-routing": "0.0.6",
+    "bootstrap-colorpicker": "^3.2.0",
```

Comment:
This could maybe be added in the UI kit, wdyt @NeOMakinG ?

Reply:
Yes it should, but it would not be up on develop until we change the uikit version on develop

Reply:
@NeOMakinG Would you please create an issue in the ui-kit project so that we don't forget?

Reply:
Done : https://github.com/PrestaShop/prestashop-ui-kit/issues/91

Sorry, didn't see your answer